docs/topics/patching-nodes.md

```diff
@@ -0,0 +1,40 @@
+# Patching Cluster Nodes
+
+To favor a consistent behavior, AKS Engine's approach to infrastructure management is immutability.
+By default, cluster nodes are not patched on bootstrap nor on a regular basis. 
+
+To prevent node patching, AKS Engine creates file `/etc/apt/apt.conf.d/99periodic` with the following content:
+
+```
+APT::Periodic::Update-Package-Lists "0";
+APT::Periodic::Download-Upgradeable-Packages "0";
+APT::Periodic::AutocleanInterval "0";
+APT::Periodic::Unattended-Upgrade "0";
+```
+
+## Enable Node Patching
+
+AKS Engine allows changes to the default node patching configuration in situations where applying the latest security patches beats the benefits of immutable infrastructure.
+
+Setting `linuxProfile.enableUnattendedUpgrades` to `"true"` configures each Linux node VM (including control plane node VMs) to run `/usr/bin/unattended-upgrade` in the background according to a daily schedule. If enabled, the default `unattended-upgrades` package configuration will be used as provided by the Ubuntu distro version running on the VM. More information [here](https://help.ubuntu.com/community/AutomaticSecurityUpdates).
+
+## Orchestrate Node Reboots
+
+To ensure that the nodes are rebooted in a non-disruptive way, you can deploy the [kured](https://github.com/weaveworks/kured) daemonset.
+
+The `kured` daemonset can be installed by following this [instructions](https://kured.dev/docs/installation/):
+
+```bash
+latest=$(curl -s https://api.github.com/repos/kubereboot/kured/releases | jq -r '.[0].tag_name')
```

Comment:
We should point out that they might not want the latest kured version if their cluster is on an older kubernetes version, for ex. K8s V1.24 would not work with latest Kured. 
(https://kured.dev/docs/installation/#kubernetes--os-compatibility)

Reply:
ok ok